Python/Product/PythonTools/PythonTools/LanguageServerClient/PythonLanguageClient.cs

```diff
@@ -137,13 +139,13 @@ internal sealed class PythonLanguageClient : ILanguageClient, ILanguageClientCus
             await JoinableTaskContext.Factory.SwitchToMainThreadAsync();
             CreateClientContexts();
 
-
+            _deferredSettingsChangedTimer = _deferredSettingsChangedTimer = new Timer(OnDeferredSettingsChanged, state: null, Timeout.Infinite, Timeout.Infinite);
```

Comment:
Why are there two `_deferredSettingsChangedTimer =`?

Reply:
thanks. that was a mistake